source/blood/src/view.cpp

```diff
@@ -1138,10 +1138,46 @@ void DrawStatNumber(const char *pFormat, int nNumber, int nTile, int x, int y, i
     int width = tilesiz[nTile].x+1;
     x <<= 16;
     sprintf(tempbuf, pFormat, nNumber);
-    for (unsigned int i = 0; i < strlen(tempbuf); i++, x += width*nScale)
+    const size_t nLength = strlen(tempbuf);
+    for (size_t i = 0; i < nLength; i++, x += width*nScale)
     {
-        if (tempbuf[i] == ' ') continue;
-        rotatesprite(x, y<<16, nScale, 0, nTile+tempbuf[i]-'0', nShade, nPalette, nStat | 10, 0, 0, xdim-1, ydim-1);
+        int numTile, numScale, numY;
+        if (tempbuf[i] == ' ')
+            continue;
+        if (tempbuf[i] == '-')
+        {
+            switch (nTile)
+            {
+            default:
+                numTile = kSBarNegative;
+                break;
+            case 2240:
```

Comment:
What is going on with case 2240? [GDX indicates ](https://gitlab.com/m210/BloodGDX/-/blob/master/core/src/ru/m210projects/Blood/Types/Hud/HudRenderer.java#L5)
```java
	public final int kBigRed = 2190;
	public final int kBigBlue = 2240;
```
I do not see a 2190 case?

Reply:
2190 would be covered with the default case. And I don't think nblood has those default defines.